library/CHART-TEMPLATE/README.md

```diff
@@ -36,17 +36,27 @@ low data modes or any other quirk that it could have.
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
 | affinity | object | `{}` |  |
+| apiKey | string | `"barqux"` |  |
 | cluster | string | `"barfoo"` |  |
 | containerSecurityContext | object | `{}` |  |
+| customAPIKeySecretKey | string | `""` |  |
+| customAPIKeySecretName | string | `""` |  |
+| customInsightsKeySecretKey | string | `""` |  |
+| customInsightsKeySecretName | string | `""` |  |
```

Comment:
aren't these deprecated?
<img width="1080" alt="Screenshot 2024-08-23 at 10 50 45" src="https://github.com/user-attachments/assets/4f9999f1-930a-4f51-a166-c2ca55b2340e">

Reply:
They are, but they are needed for StatsD.

You see it in the diff because the last PR when this was added did not added it to `vaules.yaml` and they were not templated in the readme.

Reply:
PR when this was added: newrelic/helm-charts#1402